packages/gatsby/src/bootstrap/index.js

```diff
@@ -211,6 +212,25 @@ module.exports = async (args: BootstrapArgs) => {
 
   activity.end()
 
+  // Start the nodes database (in memory loki js with interval disk
+  // saves). If data was saved from a previous build, it will be
+  // loaded here
+  activity = report.activityTimer(`start nodes db`, {
+    parentSpan: bootstrapSpan,
+  })
+  activity.start()
+  const dbSaveFile = `${program.directory}/.cache/loki/loki.db`
+  try {
+    await db.start({
+      saveFile: dbSaveFile,
+    })
+  } catch (e) {
+    report.error(
+      `Error starting DB. Perhaps try deleting ${path.dirname(dbSaveFile)}`
```

Comment:
🤔 probably worth just doing it automatically, or at least prompting?

Reply:
My thoughts too! Glad you agree. I'll implement it